
# <code>MotifPeeker-reports</code>

<img
src="https://github.com/neurogenomics/MotifPeeker/raw/master/inst/hex/hex.png"
style="height: 300px !important;" />

**Authors:** ***Hiranyamaya (Hiru) Dash, Thomas Roberts, Nathan
Skene***  
**Updated:** ***Jul-16-2024***

[`MotifPeeker-reports`](https://github.com/neurogenomics/MotifPeeker-reports/)
is a public archive for interactive HTML reports generated by
[`MotifPeeker`](https://github.com/neurogenomics/MotifPeeker), along
with the code used to produce them.

## [Reports Home](https://neurogenomics.github.io/MotifPeeker-reports/)

### Reports

#### [CTCF ChIP-seq - TIP-seq](https://neurogenomics.github.io/MotifPeeker-reports/CTCF_ChIP-TIP/output/MotifPeeker_20240711_230621/MotifPeeker.html)

<details>
<summary>
Dataset Details
</summary>

Compares three public ChIP-seq datasets against a TIP-seq dataset for
CTCF in HCT116 cells. Details of the datasets used are as follows:

1.  <code>ChIPSeq1</code>  
    ENCODE Accession:
    [ENCFF172XLC](https://www.encodeproject.org/files/ENCFF172XLC/)  
2.  <code>ChIPSeq2</code>  
    ENCODE Accession:
    [ENCFF411OOS](https://www.encodeproject.org/files/ENCFF411OOS/)  
3.  <code>ChIPSeq3</code>  
    ENCODE Accession:
    [ENCFF091ODJ](https://www.encodeproject.org/files/ENCFF091ODJ/)
4.  <code>TIPSeq</code>  
    Alignment file produced by running the
    [`nf-core/cutandrun`](https://nf-co.re/cutandrun/) pipeline on the
    raw sequencing data sourced from *NIH Sequence Read Archives* [*ID:
    SRR16963166*](https://trace.ncbi.nlm.nih.gov/Traces/index.html?view=run_browser&acc=SRR16963166&display=metadata).

Peak calling was performed using
[`MACS3`](https://github.com/macs3-project/MACS) using the following
commands:

``` r
## For ChIP-Seq datasets
MACSr::callpeak(
        tfile = ...,
        cfile = NULL,  # No control file
        nomodel = FALSE,
        qvalue = 0.01,
        format = "BAM",
        name = "chipseq",
        outdir = ...
    )
    
## For TIP-Seq dataset
MACSr::callpeak(
        tfile = tipseq_alignment_data,
        nomodel = TRUE,
        qvalue = 0.01,
        shift = -75,  # 5' shift
        extsize = 150,
        keepduplicates = "all",
        format = "BAM",
        name = "tipseq",
        outdir = ...
    )
```

Motif files used:

1.  CTCF motif [MA1930.2](https://jaspar.elixir.no/matrix/MA1930.2/)
2.  CTCFL motif [MA1102.3](https://jaspar.elixir.no/matrix/MA1102.3/)

</details>

 [Code](https://github.com/neurogenomics/MotifPeeker-reports/blob/master/CTCF_ChIP-TIP/run.R)

#### [Built-in data](https://neurogenomics.github.io/MotifPeeker-reports/built_in_data/output/MotifPeeker_20240716_104747/MotifPeeker.html)

<details>
<summary>
Dataset Details
</summary>
Report generated from built-in data, as described in the
<a href="https://neurogenomics.github.io/MotifPeeker/articles/MotifPeeker.html">
<code>MotifPeeker</code> vignette</a>.
</details>

 [Code](https://github.com/neurogenomics/MotifPeeker-reports/blob/master/built_in_data/run.R)
